README.md

```diff
@@ -20,11 +20,16 @@ Before you start, ensure you have the following installed:
 - [Postgres](https://www.postgresql.org/)
 - [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
 
+You'll also need to ask for `space-developer` permissions for the federalist cloud.gov org in the Federalist slack channel.
```

Comment:
`space-developer` -> `SpaceDeveloper`

(that's the way you have to format it in the CF CLI)